pdr_backend/cli/cli_arguments.py

```diff
@@ -112,6 +133,51 @@ def add_argument_LOOKBACK(self):
         )
 
 
+@enforce_types
+def check_positive(value):
+    ivalue = int(value)
```

Comment:
can you also wrap this line in a try/catch? If I input something that can't be converted to int, the type error will not be raised :)

Reply:
wrapped